course_info/templates/course_info/partials/details.html

```diff
@@ -4,7 +4,7 @@
     <h3 class="breadcrumbs">
         {% endverbatim %}
         <a href="{% url 'dashboard_account' %}">Admin Console</a>
-        &gt; <a href="#/">Find Course Info</a>
+        &gt; <a href="#/">Search Courses</a>
```

Comment:
Can we make all of these consistent across our tools while we're at it? I think the most recent iteration was explored in https://github.com/Harvard-University-iCommons/canvas_course_admin_tools/pull/47/files.